jboss-eap-standalone-openshift/azuredeploy.json

```diff
@@ -0,0 +1,375 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "User name for the Virtual Machine and OpenShift Webconsole."
+      }
+    },
+    "adminPassword": {
```

Comment:
remove this since you don't allow password authn

Reply:
We use this password for the OCP console. We need user to specify it. If I remove it, how will user specify password to log in to OCP console?

Reply:
changed name from adminPassword to openShiftPassword